### PR TITLE
Apply gradle-java-distribution --> sls-packaging renaming

### PR DESCRIPTION
**Pull request by uschi2000**
This change is [Reviewable](https://reviewable.io/reviews/palantir/sls-packaging/172)

### CODE REVIEW

**Comment by markelliot** [comment]
you're missing the associated META-INF resource changes?

File: build.gradle [hunk 1]
```diff
@@ -31,7 +31,7 @@ if (System.env.CIRCLE_TEST_REPORTS) {
     test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, getName())
 }
 
-group 'com.palantir.gradle.javadist'
+group 'com.palantir.gradle.sls-packaging'
```
Comment (markelliot) [inline build.gradle:34]: note that when this changes we'll need to publish to a different bintray destination and we'll need to get that whitelisted to jcenter.

**Comment by uschi2000** [comment]
Yep, aware of the bintray thing

**Comment by uschi2000** [comment]
Was going to do the meta INF thing in a separate PR. Artifact names and
plugin names are unrelated.

**Comment by markelliot** [comment]
The META-INF thing maps plugin name to implementation. If you change the plugin name, the resource also needs to change.

Sent from my iPhone

On Jan 30, 2017, at 12:50 PM, Robert Fink <notifications@github.com<mailto:notifications@github.com>> wrote:

Was going to do the meta INF thing in a separate PR. Artifact names and
plugin names are unrelated.

**Comment by uschi2000** [comment]
Ah, seems like the gradle plugin publishing was borked already after we introduced the second plugin. Any way, I reverted the name change for the plugin here.

File: build.gradle (hunk 1 above)
Comment (uschi2000) [reply build.gradle:34]: yep

**Pull request merged**